### PR TITLE
add heartbeat log

### PR DESCRIPTION
**Pull request by rgarcia**
for monitoring E2E ES delivery

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -72,9 +72,17 @@ func getHostname() string {
 	return hostname
 }
 
+func heartbeat() {
+	for _ = range time.Tick(1 * time.Second) {
```
Comment (rgarcia) [inline main.go]: @burnsed not sure how often heka used to log a heartbeat, so I just made it every second...

Comment (burnsed) [reply main.go]: I believe it was 15 seconds: https://github.com/Clever/ecs-logparser/blob/master/heka-config/log-monitor.toml#L18

**Review by burnsed — APPROVED** [review]
I'd move the heartbeat to 15sec so its not so spammy. 

Also, looks like log-monitor-es needs some updates to handle the change in case: https://github.com/Clever/log-monitor-es/blob/master/main.go#L46

File: main.go (hunk 1 above)
Comment (rgarcia) [reply main.go]: switched to 15s a46d16d

**Comment by rgarcia** [comment]
@burnsed yeah working on log-monitor-es changes, but pushing this out first will help to test

**Pull request merged**